### PR TITLE
RA-197: Feature flags + Documentation

### PR DESCRIPTION
**Pull request by JoakimHagen**
I'm not 100% happy with the template parts of the documentation. The limitations of static text impede the usability of the scripts shown. I've made effort to keep the substitution work minimal for scripts, but it may impact readability somewhat.

All three major OSes covered. Authentication with docker added. Alias options for Windows added. Help prints updated.

### CODE REVIEW

File: cmd/buildDeployApplication.go [hunk 1]
```diff
@@ -23,6 +23,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const buildDeployApplicationEnabled = true
```
Comment (ingeknudsen) [inline cmd/buildDeployApplication.go]: Should be disabled

File: cmd/follow.go [hunk 2]
```diff
@@ -21,6 +21,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const followEnabled = false
```
Comment (ingeknudsen) [inline cmd/follow.go]: Should be enabled. Kjell Erik suggested that we try to deliver on the issue described here, as well as deliver on deploy-only:
https://github.com/equinor/radix-platform/issues/225

File: cmd/get.go [hunk 3]
```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const getEnabled = true
```
Comment (ingeknudsen) [inline cmd/get.go]: Should be false

File: cmd/list.go [hunk 4]
```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const listEnabled = true
```
Comment (ingeknudsen) [inline cmd/list.go]: Should be disabled

File: cmd/set.go [hunk 5]
```diff
@@ -31,6 +33,8 @@ var setCmd = &cobra.Command{
 }
 
 func init() {
-	rootCmd.AddCommand(setCmd)
-	setCmd.AddCommand(setEnvironmentSecretCmd)
+	if setEnabled {
+		rootCmd.AddCommand(setCmd)
+		setCmd.AddCommand(setEnvironmentSecretCmd)
```
Comment (ingeknudsen) [inline cmd/set.go]: Should there not be a feature flag on setEnvironmentSecretCmd, even though we want it to be enabled? Might be "smør på flesk"...

File: README.md [hunk 6]
```diff
@@ -53,7 +101,7 @@ rx --token-environment list applications
 
 ```
   build-deploy Will trigger build-deploy of a Radix application
-  follow       Follow Radix resources
+  get          Get Radix resource
```
Comment (ingeknudsen) [inline README.md]: Maybe we should remove this block entirely, as the users can for them selves see the supported commands and if we forget the documentation might be wrong

File: cmd/set.go (hunk 5 above)
Comment (JoakimHagen) [reply cmd/set.go]: Does it make sense to enable "set" but keep the sub-command "set environment-secret" disabled? I cannot see any other settable resource. "set-context" is its own command. For now I've only implemented feature toggling for lvl1 subcommands.

**Pull request merged**